common/bitvec/src/lib.rs

```diff
@@ -213,13 +212,17 @@ mod test {
         p5.set(10);
         assert!(p5.is_set(10));
         assert_eq!(p5.last_set_bit(), Some(10));
-        assert_eq!(p5.inner, vec![0b0000_0001, 0b0110_0000])
+        assert_eq!(p5.inner, vec![0b0000_0001, 0b0110_0000]);
+
+        let p6 = BitVec { inner: vec![128u8] };
```

Comment:
let's write this as a binary literal so the test is clearer

```suggestion
        let p6 = BitVec { inner: vec![0b1000_0000] };
```